<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_add">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodanie nowego auta</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mark">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_mark">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Marka</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="model">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_model">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Model</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="year">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_nrVIN">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Numer VIN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nrVIN">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_year">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rok produkcji</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_addCar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj auto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_show_all">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Pokaz wszystkie</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="output">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_older">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Starsze od:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="older">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_older">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Policz</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_remove">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuniecie auta po numeru VIN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="remove">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_remove">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Usun</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sortRok">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>SortRok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sortMark">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>SortMark</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nazwa">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>50</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_liczba">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Liczba ludnosci</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="liczba">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_add_kraj">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodanie nowego kraju</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_nazwa">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nazwa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add_kraj">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>130</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj kraj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_show_all_kraje">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>160</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Pokaz wszystkie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_show_kraje">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>190</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Pokaz tylko kraje</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_do">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>80</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Do:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_od">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>30</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Od:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zakres_od">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>50</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="zakres_do">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>100</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_find">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyszukanie z zakersu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_find">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>130</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyszukaj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_remove_kraj">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>180</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Usun</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="remove_kraj">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>180</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_remove_kraj">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>160</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuniecie kraju po nazwie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sort_rosn">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>220</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Rosnaco</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sort_mal">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>250</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Malejaco</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1009</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
